app/models/academic_cycle.rb

```diff
@@ -18,6 +18,10 @@ def self.for_date(date)
     where("end_date >= :date AND start_date <= :date", date: date).first
   end
 
+  def self.current
+    for_date(Time.zone.now)
+  end
+
```

Comment:
There is an `AcademicYear#current?` method with a somewhat different definition. I wonder if it is worth amending that one as well?